model/metrics/database-metrics.yaml

```diff
@@ -33,6 +33,154 @@ groups:
       - ref: db.namespace
         requirement_level:
           conditionally_required: If available.
+      - ref: thread.id
```

Comment:
I'm curious about the addition of the thread.id on all of these. Do you have a use case in mind with this?

Reply:
My colleague mentioned the rationale in his comment [here](https://github.com/open-telemetry/semantic-conventions/pull/1186#discussion_r1662996947)

In short, for the case of single-threaded operations, the thread id could be useful for troubleshooting slow/blocked queries, and it might also be helpful to aggregate stats on a certain thread for diagnosing performance issues. But we're open to discussion on the attribute and dropping it from the spec if the consensus is that these use cases are too niche and/or not really practical.

Reply:
Does thread id add to the cardinality in an uncontrolled way here?